File: java/examples/book/petrol-pump/chapter2/section2/Beeper.java
package chapter2.section2;

import pump.*;
import sodium.*;

public class Beeper implements Pump
{
    public Pump.Outputs create(Pump.Inputs inputs)
    {
        return new Pump.Outputs().setBeep(
            inputs.eKeypad.map(k -> Unit.UNIT)
        );
    }
}

File: c#/Book/PetrolPump/PetrolPump/PetrolPump/Chapter2/Section3/Beeper.cs
using PetrolPump.Pump;

using Sodium;

namespace PetrolPump.Chapter2.Section3
{
  public class Beeper : IPump
  {
    public Outputs Create(Inputs inputs)
    {
      return new Outputs().SetBeep(inputs.EKeypad.Map(k => Unit.UNIT));
    }
  }
}
